editor/components/page-attributes/parent.js

```diff
@@ -0,0 +1,83 @@
+/**
+ * External dependencies
+ */
+import { connect } from 'react-redux';
+import { get } from 'lodash';
+import { stringify } from 'querystringify';
+
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import { withInstanceId, withAPIData } from '@wordpress/components';
+import { compose } from '@wordpress/element';
+import { buildTermsTree } from '@wordpress/utils';
+
+/**
+ * Internal dependencies
+ */
+import { getCurrentPostId, getEditedPostAttribute, getCurrentPostType } from '../../store/selectors';
+import { editPost } from '../../store/actions';
+import { TermTreeSelect } from '@wordpress/blocks';
+
+export function PageAttributesParent( { parent, postId, postType, items, onUpdateParent } ) {
+	const isHierarchical = get( postType, 'data.hierarchical', false );
+	if ( ! isHierarchical ) {
```

Comment:
Should we return `null` while loading the pages to avoid the showing "no parent" when loading when there's an assigned parent?

Reply:
Improved 👍